pkg/sources/git/git.go

```diff
@@ -189,8 +189,14 @@ func (s *Source) Chunks(ctx context.Context, chunksChan chan *sources.Chunk) err
 		return errors.New("invalid connection type for git source")
 	}
 
-	for i, u := range s.conn.Directories {
-		s.SetProgressComplete(i, len(s.conn.Repositories), fmt.Sprintf("Repo: %s", u), "")
+	for _, u := range s.conn.Directories {
+		_ = s.Counter.IncTotal()
```

Comment:
**Nit (optional):** Unless this causes a lint error, I think we can remove the `_ =` here. Maybe it's just me, but it looks like we're ignoring an error instead of ignoring the new value.